Makefile

```diff
@@ -238,13 +239,7 @@ release-src:
 	--exclude t \
 	--exclude utils \
 	--exclude release \
-	--exclude $(RELEASE_SRC).tgz \
```

Comment:
We already switch to a whitelist solution: https://github.com/apache/apisix/pull/3466/files

Reply:
Cool I didn't realize whitelist could solve this problem.